README.md

```diff
@@ -883,6 +883,47 @@ client.getDocuments(ids).then((docs) => {
   // [{_id: 'bike123', ...}, null, {_id: 'bike345', ...}]
 })
 ```
+### Listening to live content updates
+
+> [!NOTE]
+>
+> Live Content is experimental and require your client config to be set up with `apiVersion: 'vX'`.
+
+```ts
+// Subscribe to live updates
+const subscription = client.live.events().subscribe(
+  (event) => { 
+    // Check if incoming tags match saved sync tags 
+    if (event.type === "message" && event.tags.some((tag) => syncTags.includes(tag))) { 
+      // Refetch with ID to get latest data
+      render(event.id)
+    }
+    if (event.type === "restart") {
+      // A restart event is sent when the `lastLiveEventId` we've been given earlier is no longer usable
+      render()
+    }
```

Comment:
I'm not sure how much sense this makes out of context.  Maybe it needs to be more clear this is a rough example of subscribing (more vague), or it needs to include the surrounding context like the render function, receiving syncTags from the response (more specific)

Reply:
Yeah, I wasn't sure how much we should go into it here since the convention is to make the code examples fairly light-weight. Maybe we should just link to the docs for implementation details.

Reply:
It's still useful and should be included, the most common case where a `restart` will happen is if you lose your internet connection and are offline long enough, and the live events that happen while you are gone are many enough, that the API can't replay everything that happened. So instead of replying the events, you'll get the `restart` and should refetch as it's highly likely the content you're looking at is stale.